src/app/server/Server.h

```diff
@@ -340,6 +340,8 @@ class Server
 
     Credentials::OperationalCertificateStore * GetOpCertStore() { return mOpCertStore; }
 
+    app::DefaultAttributePersistenceProvider & GetAttributePersister() { return mAttributePersister; }
```

Comment:
Should this be called `GetDefaultAttributePersister()`, since it does not get the currently-registered one?

Reply:
Right, done.